### PR TITLE
volume x axis and chart

### PR DESCRIPTION
**Pull request by mpgxvii**
solves #302

### CODE REVIEW

File: src/app/subject/components/source-graphs/source-graphs.component.html [hunk 1]
```diff
@@ -18,35 +20,43 @@
...
 </div>
 
-<!-- <div class="bottom-group">
+<div class="bottom-group">
   <app-source-date-axis
     class="common-axis"
-    [dates]="dates"
-  ></app-source-date-axis> -->
+    [timeFrame]="sensorsDataTimeFrame"
+  ></app-source-date-axis>
```
Comment (nivemaham) [inline src/app/subject/components/source-graphs/source-graphs.component.html:53]: Nice work. When I run it on my machine, I do not see this common-axis for data. I can see the volume bar. 
I see the `timeFrame` value is null.

**Review by nivemaham — APPROVED** [review]
Nice work. LGTM .
How do we calculate the `sensorDataTimeFrame`? I thought it would be different, specially smaller timeframe than the `volumeTimeFrame`.

**Comment by mpgxvii** [comment]
`sensorDataTimeFrame` is set by the brush and zoom feature. By default, on load, `sensorDataTimeFrame` is equal to `volumeTimeFrame`. Once the user modifies the brush on the volume chart, the `sensorDataTimeFrame` is updated.

**Review by nivemaham — APPROVED** [review]
LGTM

**Pull request merged**